handlers/workloads.go

```diff
@@ -68,7 +70,13 @@ func WorkloadList(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if criteria.IncludeHealth {
-		rateInterval, err := adjustRateInterval(r.Context(), businessLayer, p.Namespace, p.RateInterval, p.QueryTime)
+		// When the cluster is not specified, we need to get it. If there are more than one, get the first one
+		clusters, _ := businessLayer.Namespace.GetNamespaceClusters(context.TODO(), p.Namespace)
+		if len(clusters) == 0 {
+			handleErrorResponse(w, err, "Error looking for cluster: "+err.Error())
+			return
+		}
+		rateInterval, err := adjustRateInterval(r.Context(), businessLayer, p.Namespace, p.RateInterval, p.QueryTime, clusters[0].Cluster)
```

Comment:
Kind of the same question here: is rate interval tied to a specific cluster or no? It seems like it would be random which cluster you got in `clusters[0].Cluster` which would make results inconsistent?

Reply:
See above, the createTimestamp should be set to the Namespace on the specific cluster, or the oldest time for all of the namespace clusters.

Reply:
Updated, thanks